services/web/server/tests/unit/conftest.py

```diff
@@ -42,12 +42,7 @@ def here() -> Path:
     return cdir
 
 
-@pytest.fixture(scope="session")
-def fake_static_dir(fake_data_dir: Path) -> Path:
-    return fake_data_dir / "static"
-
-
-@pytest.fixture(scope="session")
+@pytest.fixture
```

Comment:
forgot ``(scope="session")``

Reply:
There is no need for this to be session scoped, as it returns a string. Am I missing something important?

Reply:
not absolutely critical since, as you say, it is returning a constant. But for the same reason, why did you change the scope? 

The differences are:
Before: it calls fixture ONCE
Now: it calls the fixture AS MANY TIMES as is used

Reply:
put back to session scope